tensorflow/lite/micro/micro_profiler.cc

```diff
@@ -67,4 +67,58 @@ void MicroProfiler::LogCsv() const {
 #endif
 }
 
+void MicroProfiler::LogTicksPerTagCsv() {
+#if !defined(TF_LITE_STRIP_ERROR_STRINGS)
+  MicroPrintf(
+      "\"Unique Ops in the Graph\",\"Total Ticks (all instances of the Op)\"");
```

Comment:
nit: technically may not be Ops. It could be any event that we profile with the MicroProfiler.

Unique Tag, Total ticks across all events with that tag.